modules/rosa-create-an-identity-based-policy.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * rosa-adding-additional-constraints-for-ip-based-aws-role-assumption/rosa-create-an-identity-based-policy.adoc
+
+:_content-type: PROCEDURE
+[id="rosa-create-an-identity-based-policy_{context}"]
+= Create an Identity-based IAM policy
+
+:context: rosa-create-an-identity-based-policy
+
+toc::[]
+
+You can create an Identity-based Identity and Access Management (IAM) policy that denies access to all AWS actions when the request originates from an IP address other than Red Hat provided IPs.
+
+.Prerequisites
+
+* You have access to the AWS Management Console with the permissions required to create and modify IAM policies.
+
+.Procedure
+
+. Sign in to the AWS Management Console using your AWS account credentials.
+. Navigate to the IAM service.
+. In the IAM console, select *Policies* from the left navigation menu.
+. Click *Create policy* to begin creating a new policy.
+. In the Create policy page, select the *JSON* tab to define the policy using JSON format.
+. Copy and paste the following JSON policy document into the editor:
++
+[source,json]
+----
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Deny",
+            "Action": "*",
+            "Resource": "*",
+            "Condition": {
+                "NotIpAddress": {
+                    "aws:SourceIp": [
+                        "x.x.x.x/32",   /* Allowlisted IP 1 */
+                        "y.y.y.y/32",   /* Allowlisted IP 2 */
+                        "z.z.z.z/32"    /* Allowlisted IP 3 */
+                    ],
+                "Bool": {"aws:ViaAWSService": "false"}
+                }
+            }
+            
+        }
+
+    ]
+}
+----
++
+. Replace the x.x.x.x/32, y.y.y.y/32, and z.z.z.z/32 placeholders with the actual IP addresses you want to allow.
++
+[NOTE]
+====
+You must specify the IP addresses in CIDR notation.
+====
++
+. Click *Review policy*.
```

Comment:
The AWS console UI is changed, there is no 'Review policy' element on the step 'Review and create' on the 'Create policy' page